test/e2e/utils/tls.go

```diff
@@ -29,7 +31,7 @@ const (
 	tmpDir             = "/tmp/infinispan/operator/tls"
 )
 
-var serialNumber int64 = 1
+var serialNumber int64 = mrand.Int63n(math.MaxInt64)
```

Comment:
You need to either seed  mrand or use the "crypto/rand" package here. Otherwise the randomness is deterministic on each process start, which we don't want in this case as we're trying to avoid two simultaneous  test runs having the same `serialNumber`.

https://stackoverflow.com/questions/63655205/random-integer-in-go-language-are-predictable-in-each-runtime

Reply:
Fixed, hopefully correctly